config/locales/idv/en.yml

```diff
@@ -185,7 +185,7 @@ en:
         review: Re-enter your %{app_name} password to protect your data
     troubleshooting:
       headings:
-        are_you_near: 'Are you located near Washington, D.C. <br/>or Baltimore, MD?'
+        are_you_near: 'Are you located near Washington,<nbsp/>D.C. <wbr/>or<nbsp/>Baltimore,<nbsp/>MD?'
```

Comment:
I wonder if we could just insert the non-breaking space directly?

```suggestion
        are_you_near: 'Are you located near Washington, D.C. or Baltimore, MD?'
```

It seems to get past the YAML normalization without issue. I think it does pose a risk to be confusing to developers since visually it appears the same as a normal space, though at least in VSCode there is some additional highlighting on the irregular whitespace:

![image](https://user-images.githubusercontent.com/1779930/193601617-8bbb8996-58eb-4ae2-91f1-101e74781940.png)